x/erc20/keeper/str_v2_test.go

```diff
@@ -0,0 +1,50 @@
+package keeper_test
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (suite *KeeperTestSuite) TestGetAllSTRV2Address() {
+	address1 := suite.address.Bytes()
+	address2 := suite.consAddress.Bytes()
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expected []sdk.AccAddress
+	}{
+		{
+			"space is empty",
+			func() {
+			},
+			[]sdk.AccAddress{},
+		},
+		{
+			"one address",
+			func() {
+				suite.app.Erc20Keeper.SetSTRv2Address(suite.ctx, address1)
```

Comment:
Nit: add test cases for other functions:
- SetSTRv2Address
- to make sure SetSTRv2Address is idempotent if trying to set an address that aldready exists
- HasSTRv2Address
- DeleteSTRv2Address